java/pom.xml

```diff
@@ -61,6 +61,12 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>com.google.protobuf</groupId>
+        <artifactId>protobuf-bom</artifactId>
+        <version>${project.version}</version>
+        <type>pom</type>
+      </dependency>
```

Comment:
I believe this also needs to have `<scope>import</scope>`

Reply:
That fixed it. Thanks. PTAL @garrettjonesgoogle @ejona86